src/components/layout/SidebarLinks.tsx

```diff
@@ -33,6 +33,7 @@ const renderLink = (
         w="140px"
         my="14px"
         key={link.key}
+        className="hide-on-mobile"
```

Comment:
why do you use `.hide-on-mobile` and use Mantine breakpoint with `display` to do that instead in `src/components/layout/Shell.tsx`? It seems you're solving the same problem in multiple ways. Can we use only one instead of two?

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/1476534d34545c4a1c3076663a715200ad25e5aa to use `hide-on-mobile` consistently.